bindata/network/network-metrics/002-daemonset.yaml

```diff
@@ -0,0 +1,74 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: network-metrics-daemon
+  namespace: openshift-network-metrics
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the network metrics daemon on each node
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+    networkoperator.openshift.io/non-critical: ""
+spec:
+  selector:
+    matchLabels:
+      app: network-metrics-daemon
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: network-metrics-daemon
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      tolerations:
+        - operator: Exists
+      containers:
+        - name: network-metrics-daemon
```

Comment:
Doesn't this container need the pod to have hostNetwork: true?

Reply:
Nope, at the moment it listens for pods and publish metrics related to the secondary network names